source/SnapDump-Handler/SDBasicHTTPStore.class.st

```diff
@@ -8,3 +8,9 @@ Class {
 	],
 	#category : #'SnapDump-Handler'
 }
+
+{ #category : #testing }
+SDBasicHTTPStore class >> storeName [ 
```

Comment:
I need to check again. But I unified Basic http store and http store. I thought the basic one I removed

Reply:
I see indeed that this class is empty, there is no references and that HTTPStore is part of the core package.
Maybe the class definition BasicHTTPStore has somehow remained in between 2 changes by mistake?
Should I remove it completely ?

Reply:
I think so. I wanted to have to smallest possible code one needs to load in order to use SnapDump. But as you guys introduced JSON and such I decided that the separation of basic http handler and client is not needed anymore. Maybe I forgot to delete the class.